platform/core/PolyPodCoreAndroid/core/src/main/kotlin/coop/polypoly/core/UpdateNotification.kt

```diff
@@ -0,0 +1,21 @@
+package coop.polypoly.core
```

Comment:
Is this just a move? Did you use `git mv` here to keep history?

Reply:
There's no need to use `git mv`, Git figures out renames based on how similar two objects are. The interface is still roughly the same, but the implementation is entirely different now.

Reply:
Maybe, but not GitHub. As you see, it shows up as a file deleted above, and another that's popped up here. 
![Captura de pantalla de 2022-10-19 09-20-56](https://user-images.githubusercontent.com/500/196623409-ce6c09e3-5a2c-4929-8855-8b26e3052458.png)
Not such a big deal, and history is kept if you really look hard. It's just convenient for reviews is all. And not so different to do git mv instead of mv.